appengine/Rakefile

```diff
@@ -0,0 +1,19 @@
+$VERBOSE = true
```

Comment:
Leftover from debugging. Removed in https://github.com/GoogleCloudPlatform/ruby-docker/pull/24